### ISSUE TITLE
Check channel password on reconnect

### ISSUE THREAD

**Issue by Madasanya** [GQDeltex/ft_transcendence#204]
when a user reconnects, it should also check that their channel passwords are still valid and that it hasn't been changed

if not, channelUser should be deleted.

**Comment by GQDeltex** [comment]
I don't think this is useful. If I am already in the channel, why would I be kicked if the owner changes the password. I think it's enough if you can't join again. If the owner (or admins) don't want you in the channel they can simply kick you.
